File: Datas/本地数据表/CreateLua/ItemEntity.lua
ItemEntity = { Id = 0, Name = "", Type = 0, UsedLevel = 0, UsedMethod = "", SellMoney = 0, Quality = 0, Description = "", UsedItems = "", maxAmount = 0, packSort = 0 }

--这句是重定义元表的索引，就是说有了这句，这个才是一个类
ItemEntity.__index = ItemEntity;

function ItemEntity.New(Id, Name, Type, UsedLevel, UsedMethod, SellMoney, Quality, Description, UsedItems, maxAmount, packSort)
    local self = { }; --初始化self
    setmetatable(self, ItemEntity); --将self的元表设定为Class

    self.Id = Id;
    self.Name = Name;
    self.Type = Type;
    self.UsedLevel = UsedLevel;
    self.UsedMethod = UsedMethod;
    self.SellMoney = SellMoney;
    self.Quality = Quality;
    self.Description = Description;
    self.UsedItems = UsedItems;
    self.maxAmount = maxAmount;
    self.packSort = packSort;

    return self;
end

File: Assets/YouYouScript/Data/DataTable/Create/ItemEntity.cs

//===================================================
//作    者：边涯  http://www.u3dol.com
//创建时间：2019-11-07 20:46:04
//备    注：此代码为工具生成 请勿手工修改
//===================================================
using System.Collections;
using YouYouFramework;

/// <summary>
/// Item实体
/// </summary>
public partial class ItemEntity : DataTableEntityBase
{
    /// <summary>
    /// 名称
    /// </summary>
    public string Name;

    /// <summary>
    /// 道具类别
    /// </summary>
    public int Type;

    /// <summary>
    /// 使用等级
    /// </summary>
    public int UsedLevel;

    /// <summary>
    /// 使用方法
    /// </summary>
    public string UsedMethod;

    /// <summary>
    /// 售价
    /// </summary>
    public int SellMoney;

    /// <summary>
    /// 品质
    /// </summary>
    public int Quality;

    /// <summary>
    /// 描述
    /// </summary>
    public string Description;

    /// <summary>
    /// 道具使用后获得的内容
    /// </summary>
    public string UsedItems;

    /// <summary>
    /// 最大堆叠数量
    /// </summary>
    public int maxAmount;

    /// <summary>
    /// 背包陈列顺序
    /// </summary>
    public int packSort;

}
